src/linemod_detect.cpp

```diff
@@ -280,11 +280,13 @@ struct Detector: public object_recognition_core::db::bases::ModelReaderBase {
         if (!(*use_rgb_))
         {
           //add a depth-based gray image to the list of sources for matching
-          depth_->convertTo(display, CV_8U, 255.0/(1800.0));
+          depth_->convertTo(display, CV_8U, 255.0/1.8);
```

Comment:
for 16U, it is True we need different  scale factors in this case.

But we never get 16U now, since the input image now is always 32F
On Apr 1, 2015 11:06 PM, "Vincent Rabaud" <notifications@github.com> wrote:

> In src/linemod_detect.cpp
> <https://github.com/wg-perception/linemod/pull/12#discussion_r27611668>:
>
> > @@ -280,11 +280,13 @@ struct Detector: public object_recognition_core::db::bases::ModelReaderBase {
> >          if (!(*use_rgb_))
> >          {
> >            //add a depth-based gray image to the list of sources for matching
> > -          depth_->convertTo(display, CV_8U, 255.0/(1800.0));
> > +          depth_->convertTo(display, CV_8U, 255.0/1.8);
>
> but that's only if the input depth is float right ? What if it is uint16 ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wg-perception/linemod/pull/12/files#r27611668>.
>